components/index.js

```diff
@@ -46,7 +46,9 @@ class FormComponent extends Component {
     const { name, options: { classes, required = true } = {} } = this
     const { [name]: { title = this.title, hint = this.hint } = {} } = config
     const isOptional = required === false
-    const label = (title || name) + (isOptional ? ' (optional)' : '')
+    const titleLabel = title === '' ? '' : (title || name)
+    const optionalLabel = isOptional ? ' (optional)' : ''
+    const label = titleLabel + optionalLabel
```

Comment:
I don't think this should be supported - the design system mandates that inputs must have a visible label: https://design-system.service.gov.uk/components/text-input/#label-text-inputs

Reply:
You're right, the situation I was trying to replicate in the service I was working from was using the label as the heading. According to the GDS documents: 

If you’re asking just one question per page as recommended, you can set the contents of the <label> as the page heading.

I'll remove the empty label and check if setting as a heading is possible in the `hapi-govuk-question-page` plungin, and add the feature if not.

Reply:
The plugin does this automatically if there is only one form component on the page, so there is no need for blank labels.